pavelib/servers.py

```diff
@@ -240,7 +240,7 @@ def update_db():
     """
     settings = getattr(options, 'settings', DEFAULT_SETTINGS)
     for system in ('lms', 'cms'):
-        sh(django_cmd(system, settings, 'migrate', '--traceback', '--pythonpath=.'))
+        sh(django_cmd(system, settings, 'migrate', '--fake-initial', '--traceback', '--pythonpath=.'))
```

Comment:
Yes - if it's enabled, the `syncdb` skips migrations for pre-existing tables. Seems right initially, but not beyond that point.